src/In.ProjectEKA.OtpService/Notification/NotificationService.cs

```diff
@@ -0,0 +1,36 @@
+namespace In.ProjectEKA.OtpService.Notification
+{
+    using System.Threading.Tasks;
+    using Newtonsoft.Json.Linq;
+
+    public class NotificationService : INotificationService
+    {
+        private readonly INotificationWebHandler notificationWebHandler;
+
+        public NotificationService(INotificationWebHandler notificationWebHandler)
+        {
+            this.notificationWebHandler = notificationWebHandler;
+        }
+
+        public async Task<NotificationResponse> SendNotification(NotificationMessage notificationMessage)
+        {
+            return notificationMessage.NotificationAction switch
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace this 'switch' expression with a ternary conditional operator to increase readability.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5101780)